test/e2e_test_pkcs11.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2021 The Sigstore Authors.
```

Comment:
This should be 2024

Reply:
also update the line 1 to `#!/usr/bin/env bash`

thanks